app/javascript/article-form/components/Body.jsx

```diff
@@ -0,0 +1,38 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import Textarea from 'preact-textarea-autosize';
+import { Toolbar } from './Toolbar';
+
+export const Body = ({
```

Comment:
This class name is too generic as well, and from a DOM standpoint, it looks confusing. `<Body />` vs. `<body />`. Consider a more specific name.

Reply:
Renamed to `EditorBody`.